manual_scripts/active_directory/set-password-policy.ps1

```diff
@@ -1,6 +1,7 @@
 # Policy taken from Datamart prod
+Set-ADFineGrainedPasswordPolicy -Identity CustomerPSO-03 `
```

Comment:
Nitpick should we give it a different name, I assume `CustomerPSO-03` doesn't mean anything?

Reply:
These are created by AWS https://docs.aws.amazon.com/directoryservice/latest/admin-guide/supportedpolicysettings.html

Reply:
:+1: maybe worth a comment with a link then